<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMAMSUserSettingsWidget</class>
 <widget class="QWidget" name="QMAMSUserSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Benutzer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableView" name="tvUser">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="pbAddUser">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Hinzufügen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbRemoveUser">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Löschen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbChangePassword">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Passwort ändern</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbChangeUsername">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Nutzername ändern</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbChangeName">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Name ändern</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbResetLoginNum">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Anmeldungen zurücks.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbChangeActiveState">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Aktivieren</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbChangeAdminState">
            <property name="text">
             <string>Admin-Flag aktivieren</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbGroups">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Gruppenzugehörigkeit des selektierten Benutzers</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Zugewiesene Gruppen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="lvUserGroup">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbAddGroup">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Hinzufügen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbRemoveGroup">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Löschen</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Verfügbare Gruppen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="lvGroup">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbAddGroup</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSUserSettingsWidget</receiver>
   <slot>addGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAddUser</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSUserSettingsWidget</receiver>
   <slot>addUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbRemoveGroup</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSUserSettingsWidget</receiver>
   <slot>removeGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbRemoveUser</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSUserSettingsWidget</receiver>
   <slot>removeUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbChangePassword</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSUserSettingsWidget</receiver>
   <slot>changePassword()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1084</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbChangeName</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSUserSettingsWidget</receiver>
   <slot>changeName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbChangeUsername</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSUserSettingsWidget</receiver>
   <slot>changeUsername()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbResetLoginNum</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSUserSettingsWidget</receiver>
   <slot>resetLoginNum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbChangeActiveState</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSUserSettingsWidget</receiver>
   <slot>changeActiveState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbChangeAdminState</sender>
   <signal>clicked()</signal>
   <receiver>QMAMSUserSettingsWidget</receiver>
   <slot>changeAdminState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addUser()</slot>
  <slot>removeUser()</slot>
  <slot>configUser()</slot>
  <slot>addGroup()</slot>
  <slot>removeGroup()</slot>
  <slot>configGroup()</slot>
  <slot>changePassword()</slot>
  <slot>changeUsername()</slot>
  <slot>changeName()</slot>
  <slot>resetLoginNum()</slot>
  <slot>changeActiveState()</slot>
  <slot>changeAdminState()</slot>
 </slots>
</ui>
